plugin/evm/message/handler.go

```diff
@@ -6,27 +6,10 @@ package message
 import (
 	"context"
 
-	"github.com/ethereum/go-ethereum/log"
-
 	"github.com/ava-labs/avalanchego/ids"
 )
 
-var (
-	_ GossipHandler  = NoopMempoolGossipHandler{}
-	_ RequestHandler = NoopRequestHandler{}
-)
-
-// GossipHandler handles incoming gossip messages
-type GossipHandler interface {
-	HandleEthTxs(nodeID ids.NodeID, msg EthTxsGossip) error
-}
-
-type NoopMempoolGossipHandler struct{}
-
-func (NoopMempoolGossipHandler) HandleEthTxs(nodeID ids.NodeID, msg EthTxsGossip) error {
-	log.Debug("dropping unexpected EthTxsGossip message", "peerID", nodeID)
-	return nil
-}
+var _ RequestHandler = NoopRequestHandler{}
```

Comment:
nit to reduce git diffs

```suggestion
var (
	_ RequestHandler = NoopRequestHandler{}
)
```